Source/WebCore/platform/network/soup/ResourceRequestSoup.cpp

```diff
@@ -140,6 +140,18 @@ void ResourceRequest::updateSoupMessageBody(SoupMessage* soupMessage, BlobRegist
 
 }
 
+GRefPtr<GBytes> ResourceRequest::createBodyBytes() const
+{
+    auto formData = httpBody();
+    if (!formData || formData->isEmpty())
+        return nullptr;
+
+    auto bytes = formData->resolveBlobReferences()->flatten();
+    if (!bytes.size())
+        return nullptr;
+    return adoptGRef(g_bytes_new_static(bytes.data(), bytes.size()));
```

Comment:
We can't use `new_static` here, the Vector will be freed when returning from this function. We need to take a reference of FormData and use `new_with_free_func` to release the reference.